etl/steps/data/garden/gcp/2022-11-11/global_carbon_budget.meta.yml

```diff
@@ -287,8 +287,8 @@ tables:
         description: "Annual production-based emissions of carbon dioxide (CO₂), including land-use change, measured in tonnes per person. This is based on territorial emissions, which do not account for emissions embedded in traded goods."
       emissions_total_including_land_use_change_per_gdp:
         title: "Annual CO₂ emissions including land-use change per GDP"
-        unit: "tonnes"
-        short_unit: "t"
+        unit: "kilograms per $PPP"
```

Comment:
should "$PPP" be "international $" here and elsewhere, for consistency with how we refer to international dollars elsewhere on the site? e.g. https://ourworldindata.org/grapher/gdp-per-capita-worldbank (https://owid.cloud/admin/variables/446390)